src/worker/main.ts

```diff
@@ -30,7 +30,12 @@ function measureElapsedTime(job: Queue.Job, tags) {
 
 const queueOpts: QueueOptions = {
 	defaultJobOptions: {
-		attempts: 3,
+		attempts: 5,
+		timeout: 60000,
+		backoff: {
+			type: "exponential",
+			delay: 3 * 60 * 1000
+		},
```

Comment:
can you add a feature flag around this change? Better safe than sorry

Reply:
I don't think it is feature flaggable, it can be only set on startup